docs/marionette.regionmanager.md

```diff
@@ -103,7 +103,7 @@ var rm = new Marionette.RegionManager();
 var regions = rm.addRegions({
   main: '#main-content',
   navigation: {
-    selector: '#navigation',
```

Comment:
hmm we might want to deprecate it on `master`, but on `next` wouldn't it be removed entirely? If we find people were using `RegionManager` directly, which seems pretty unlikely, we can always spin up a separate module.  The interface `RegionManager` provided is now baked into `View`